src/core/client/framework/lib/relay/useRefetch.ts

```diff
@@ -14,14 +14,15 @@ type IsRefetching = boolean;
  */
 export default function useRefetch<V = Variables>(
   relay: RelayPaginationProp,
-  variables: V = {} as any
+  totalCount: number,
```

Comment:
`totalCount` here is confusing, as it always seems to be hard set to `5` in a lot of places. Using `count` might be more appropriate?

Reply:
I noticed that the underlying relay API uses `totalCount`... So approving!